src/person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :age, :name
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] Hi @Diegogagan2587, this method should not be private since it'll probably be accessible out of this class in future. You're kindly reminded to make it public :+1: